Workmode: Update with proposed new process (#48)
w3ctag/tag.w3.org: workmode/index.md

The TAG is a service organization. It exists to help ensure that the web makes sense as a platform, that the bits fit together in natural ways, and that the design is coherent. The TAG serves as an advocate for developer interests and consistent design principles.

### 1. Specification Design Reviews

> The TAG focuses primarily on providing technical feedback to people designing APIs and looking to evolve existing designs. That tends to take the form of collaborative specification review. These reviews start with a discussion between TAG members and spec authors, sometimes on a call or face-to-face meeting, and result in written feedback to the design.

Reviewer: ```suggestion
The TAG focuses primarily on providing technical feedback to people designing APIs and looking to evolve existing designs. 
That tends to take the form of collaborative specification review. 
These reviews start with a discussion between TAG members and spec authors, sometimes on a call or face-to-face meeting, and result in written feedback to the design.
```

(if we're changing this, we may as well do proper line breaking)

The TAG is a service organization. It exists to help ensure that the web makes sense as a platform, that the bits fit together in natural ways, and that the design is coherent. The TAG serves as an advocate for developer interests and consistent design principles.

### 1. Specification Design Reviews

The TAG focuses primarily on providing technical feedback to people designing APIs and looking to evolve existing designs. 

> That tends to take the form of collaborative specification review. 

Reviewer: ```suggestion
That tends to take the form of collaborative design review of proposals and/or specifications. 
```

These reviews start with a discussion between TAG members and spec authors, sometimes on a call or face-to-face meeting, and result in written feedback to the design.

The TAG is happy to provide feedback on nearly any Web-related specification, including work happening outside the W3C. To get TAG feedback, open a review issue in our [design-reviews repository](https://github.com/w3ctag/design-reviews).

> Before opening a review request, you should read:

Reviewer: ```suggestion
Before opening a design review request, you should read:
```

* [Web Platform Design Principles](https://www.w3.org/TR/design-principles/)
* [Self-Review Questionnaire: Security and Privacy](https://www.w3.org/TR/security-privacy-questionnaire)
* [What is an Explainer?](/explainers/)

We also suggest you read the [Ethical Web Principles](https://www.w3.org/2001/tag/doc/ethical-web-principles/) which forms the basis for some of the actionable advice we provide in the Web Platform Design Principles and the Security & Privacy Questionnaire.

A positive review doesn't mean that a spec is "supported" by the TAG or that the design is free of issues, but hopefully it'll help improve the designs of APIs that request feedback.

When a specification is being worked on actively in a W3C working group, please ensure that the group chair has agreed to request a TAG review before requesting it.

> [Read this page for more information about the process we follow when conducting design reviews](design-reviews).

Reviewer: ```suggestion
[Read this page for more information about our design review process](design-reviews).
```
